packages/trace-viewer/src/ui/metadataView.tsx

```diff
@@ -25,6 +25,12 @@ export const MetadataView: React.FunctionComponent<{
   if (!model)
     return <></>;
   return <div className='metadata-view vbox'>
+    {model.baseURL && (
```

Comment:
I'd move the `Config` section between `Browser` and `Viewport`. WDYT?

Reply:
yes! done in https://github.com/microsoft/playwright/pull/31852/commits/742ae72fed4939a00e0a01962aa3ce316b3f5956